tests/test_aggregates.py

```diff
@@ -357,6 +357,76 @@ def setUp(self):
             fec_election_year=2018,
         )
 
+        # Create data for future presidential - 2020. Use formula for future
```

Comment:
This could be a good opportunity to try out [Hypothesis](https://hypothesis.readthedocs.io/en/latest/) and seeing how well it pairs with [`py.test`'s parameterized functions](https://docs.pytest.org/en/latest/parametrize.html)!  I know there is a desire to get this out sooner rather than later, but I think these tests and this setup would make for an excellent candidate in trying this out in the future. :-)

Reply:
Thanks @ccostino! I forgot I put this note in here. I was thinking of using a `get_current_cycle() + 2` but it looks like that function is only in the CMS.